tests/snappi_tests/metrics_utils/metrics_reporter.py

```diff
@@ -0,0 +1,60 @@
+# This file defines the classes reporting metrics and records to the corresponding database.
+
+import logging
+import json
+import datetime
+import time
+from pprint import pprint
+from typing import List, Dict, Union
+
+#from metrics_accepter import Metric, GaugeMetric
+
+class MetricReporterFactory:
```

Comment:
move factory to a dedicated file. for reporter, we can leave in this file or move to metrics.py, no strong opinion in that.